operator/iamkeycontroller/update.go

```diff
@@ -0,0 +1,16 @@
+package iamkeycontroller
+
+import (
+	"context"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+)
+
+// Update implements managed.ExternalClient.
+// exoscale.com does not allow any updates on IAM keys.
+func (p *IAMKeyPipeline) Update(ctx context.Context, mg resource.Managed) (managed.ExternalUpdate, error) {
+	log := controllerruntime.LoggerFrom(ctx)
+	log.Info("Updating resource (noop)")
```

Comment:
Since it's a noop, I would not log it until in debug mode, it would be useless noise.
```suggestion
	log.V(1).Info("Updating resource (noop)")
```